src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -367,6 +369,35 @@ bool isWMAbsolutePositionGood()
                        std::end(wmAbsPosGood),
                        [&](const sf::String& name) { return name == windowManagerName; });
 }
+
+// Initialize raw mouse input
+void initRawMouse(::Display* disp)
```

Comment:
```suggestion
bool initRawMouse(::Display* disp)
```

Might be useful to propagate success/failure to the caller.